pkg/engine2/operational_eval/vertex_property.go

```diff
@@ -18,21 +18,17 @@ type (
 	propertyVertex struct {
 		Ref construct.PropertyRef
 
-		Template  knowledgebase.Property
-		EdgeRules map[construct.SimpleEdge][]knowledgebase.OperationalRule
+		Template      knowledgebase.Property
+		EdgeRules     map[construct.SimpleEdge][]knowledgebase.OperationalRule
+		ResourceRules map[string][]knowledgebase.OperationalRule
```

Comment:
What are these for? Resource rules that set a property should be in the property itself, no?

Reply:
i think this was before i removed configuration from additional rules, so will remove this, its just leftover